src/main/java/org/fundacionjala/coding/fizzbuzz/Kata.java

```diff
@@ -0,0 +1,46 @@
+package org.fundacionjala.coding.fizzbuzz;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+/**
+ * Write a program that prints the numbers from 1 to 100. But for multiples of three print Fizz instead
+ * of the number and for the multiples of five print Buzz. For numbers which are multiples of both three
+ * and five print FizzBuzz.
+ */
+public final class Kata {
+    /**
+     * Empty constructor.
+     */
+    private Kata() {
+    }
+
+    /**
+     * @param number quantity of print numbers
+     * @return The whole String
+     */
+    public static String fizzBuzz(final int number) {
+        final int fizz = 3;
+        final int buzz = 5;
+        int[] numbers = IntStream.rangeClosed(1, number).toArray();
+        StringBuilder result = new StringBuilder();
+        for (int a : numbers) {
+            if (a % fizz != 0 && a % buzz != 0) {
+                result.append(a + ",");
+            } else {
+                if (a % fizz == 0) {
+                    result.append("Fizz");
+                }
+                if (a % buzz == 0) {
+                    result.append("Buzz");
+                }
+                result.append(",");
+            }
+        }
+
+        return Arrays.stream(result.toString().split(","))
+                .map(e -> e.toString())
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') "e" is already a string, there's no need to call "toString()" on it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1858)